### PR TITLE
strings: add separate text for bridges and tonnels

### PR DESCRIPTION
**Pull request by alatarum**
Show Bridge and Tonnel labels for this kind of objects.

Signed-off-by: Andrei Kartashev <a.s.kartashev@gmail.com>

### CODE REVIEW

**Comment by pastk** [comment]
Thanks a lot for the PR!!!

Some translations look too long e.g. `en = Bridge with Cycle Path`/`ru = Мост с велодорожкой` compare to the `Pedestrian Bridge` / `Пешеходный мост` which is nice.

Overall I lean towards naming all the bridges just `Bridge` and similar for tunnels. The type of the bridge (bridleway, motorway, etc.) is available from the map context.
(except maybe `Pedestrian Bridge` / `Пешеходный мост` which is a very nice and natural translation)

If we agree on this then we can avoid duplicating same `Bridge`/`Мост` translation for all types - just change the refs to point to one bridge/tunnel type e.g. `[type.highway.path.bridge]`.

**Comment by alatarum** [comment]
I thought about this but failed to decide which of them should be reference one: type.highway.path.bridge, type.highway.primary.bridge, type.highway.road.bridge, etc - they all looks to be the same level ). But yes, we can take path as main one.

Other thing is should I change "living_street" and "xxx_link"? I decide to keep the translation as is for this elements, but probably they also should be changed to "Bridge".

**Comment by pastk** [comment]
Yeap highway link bridges/tunnels should be changed as well, and living_street too.

Re using references I have one doubt actually.
We usually use them to point to the parent type in cases where children types don't need specific translations or translations are not done yet.
But this case is different - we want to avoid duplication, which is good, but pointing e.g. primary_road bridge to the path bridge could have unwanted side effects. E.g. in case someone changes translation for the path bridge it will also change for the primary road bridge which is not what we might want...

**Comment by pastk** [comment]
> Yep, that is why I asked you about if we can add something like "type.default.bridge" in tg. Would be nice to have such fallback.

The biggest downside of this approach is that it won't be possible to "unref" the file easily (e.g. if we migrate to weblate) as the "type.default.bridge" would remain there as unused definition. Overall it seems to adds extra complexity for a quite small change in strings we want to do.

My suggestion is to:
1. add Bridge and Tunnel translations for all languages to existing `[type.highway.road.bridge/tunnel]`
2. add there a comment like "These translations are used for all type.highway.* bridges"
(to make translators aware of it being referenced)
3. add refs from all other highway bridges/tunnels

@biodranik  what do you think?

**Comment by biodranik** [comment]
> @biodranik what do you think?

ok

File: data/strings/types_strings.txt [hunk 1]
```diff
@@ -8244,12 +8256,71 @@
     zh-Hant = 道路
 
   [type.highway.road.bridge]
-    ref = type.highway.road
+    comment = These translations are used for all type.highway.*.bridge.
+    en = Bridge
```
Comment (pastk) [inline data/strings/types_strings.txt:8260]: Please add also
be,bg,es-MX,he,sw

File: data/strings/types_strings.txt [hunk 2]
```diff
@@ -8244,12 +8256,71 @@
...
+    th = สะพาน
+    tr = Köprü
+    uk = Міст
+    vi = Cầu
+    zh-Hans = 桥
+    zh-Hant = 橋
 
   [type.highway.road.tunnel]
+    comment = These translations are used for all type.highway.*.tunnel.
     ref = type.highway.road
```
Comment (pastk) [inline data/strings/types_strings.txt:8293]: please remove too

**Review by pastk — CHANGES_REQUESTED** [review]

File: data/strings/types_strings.txt (hunk 2 above)
Comment (alatarum) [reply data/strings/types_strings.txt:8293]: ooups

**Review by pastk — APPROVED** [review]

**Comment by pastk** [comment]
Looks great!

**Review by biodranik — APPROVED** [review]

**Comment by pastk** [comment]
Ready to merge

**Review by biodranik — APPROVED** [review]

**Pull request merged**